Workbooks/HDInsight/Overview/Overview.workbook

```diff
@@ -0,0 +1,442 @@
+{
+  "version": "Notebook/1.0",
+  "items": [
+    {
+      "type": 1,
+      "content": {
+        "json": "# HDInsight Kafka Insights"
+      },
+      "name": "text - 1"
+    },
+    {
+      "type": 9,
+      "content": {
+        "version": "KqlParameterItem/1.0",
+        "query": "",
+        "crossComponentResources": [
+          "{Workspace}"
+        ],
+        "parameters": [
+          {
+            "id": "741603f6-06d5-431c-b989-7e53c252cda7",
+            "version": "KqlParameterItem/1.0",
+            "name": "Subscription",
+            "type": 6,
+            "isRequired": true,
+            "value": "/subscriptions/d66b1168-d835-4066-8c45-7d2ed713c082",
+            "typeSettings": {
+              "additionalResourceOptions": [],
+              "includeAll": true
+            }
+          },
+          {
+            "id": "a810dcd0-f912-4926-9317-a20f24c7c5ce",
+            "version": "KqlParameterItem/1.0",
+            "name": "Workspace",
+            "type": 5,
+            "isRequired": true,
+            "query": "where type =~ 'microsoft.operationalinsights/workspaces'\r\n| project id",
```

Comment:
instead of making users pick workspaces, can you use resource-centric queries where the user picks the hdinsights resource instead?

Reply:
HDInsight is a HOBO service so we don't have resource-level platform metrics. All of the metrics are pushed to a log analytics workspace using the OMS agent, so we need the user to select the workspace where all of the metrics are being sent for their clusters.